src/app/flashcard/components/pages/flashcard-page/flashcard-page.component.ts

```diff
@@ -10,17 +12,39 @@ import { FlashcardService } from '../../../service/flashcard.service'
 })
 export class FlashcardPageComponent implements OnInit {
 
-  flashcard: Flashcard | null = null;
+  public flashcards!: Flashcard[];
 
-  constructor(flashcardService: FlashcardService) {
-    console.log("CONSTRUCTOR");
+  constructor(private flashcardService: FlashcardService) {
+    console.log('CONSTRUCTOR');
+    this.getAllFlashcards();
+  }
 
-    flashcardService.get(1).toPromise().then(res => this.flashcard = res);
+  getAllFlashcards(): void {
+    this.flashcardService.getAll().subscribe(
+      (response: any) => {
+        console.log(response.entries);
+        this.flashcards = response.content;
+      },
+      (error: HttpErrorResponse) => {
+        alert(error.message);
+      }
+    );
+
+    // this.flashcardService.getAllByDifficulty(1).subscribe(
```

Comment:
For now, remove this commented block. If we still need access to it we can look through previous commits